test/integration-test-setup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+case "$1" in
+    setup)
+        if [[ -f /inprogress ]]; then
```

Comment:
Rather than creating this in the root filesystem, it would be better to use RuntimeDirectory= or StateDirectory= so that its lifecycle is handled in the usual way, and doesn't need to be done manually, given this is running as part of a unit